types/react-instantsearch-core/index.d.ts

```diff
@@ -171,13 +172,21 @@ export function connectHighlight(Composed: React.ComponentType<any>): React.Comp
  *
  * https://community.algolia.com/react-instantsearch/connectors/connectHits.html
  */
-export function connectHits<TProps extends { hits: THit[]}, THit>(ctor: React.ComponentType<TProps>): ConnectedComponentClass<TProps, {hits?: THit[]}>;
+export function connectHits<TProps extends { hits: THit[] }, THit>(ctor: React.ComponentType<TProps>): ConnectedComponentClass<TProps, { hits?: THit[] }>;
 
 export function connectHitsPerPage(Composed: React.ComponentType<any>): React.ComponentClass<any>;
-export function connectInfiniteHits(Composed: React.ComponentType<any>): React.ComponentClass<any>;
+
+export interface InfiniteHitsProvided<THit = any> {
+  hits: THit[];
+  refine: (...args: any[]) => any;
+  hasMore: boolean;
```

Comment:
Thanks for the PR!  Can you please add the documentation for these properties from their API documentation? https://community.algolia.com/react-instantsearch/connectors/connectInfiniteHits.html